### ISSUE TITLE
Delete finished job in show action

### ISSUE THREAD

**Issue by icapuccio** [Wolox/async-requests#18]
Currently, the `async_request_jobs` table gets bigger and bigger with each async request. Since the client stops asking for a job once it is finished, it would be nice to have an automatic depuration in the table.

Another issue with the current behavior is that when you queue a job with a sensible parameter (like a password) it remains accessible for anyone that query the table.

I believe that this whole issue could be solved by performing a job destroy inside `JobsController#show` --> `render_finished_job(job)`

**Comment by santyalcala** [comment]
https://github.com/Wolox/async-requests/pull/20
